cogs/OxfordDictionary.py

```diff
@@ -8,99 +10,67 @@
 
 
 async def request(word: str) -> bs4.BeautifulSoup:
-    """
```

Comment:
Please do not remove docstrings entirely, leave them or update them to reflect changes